test/torch/linalg/test_linalg.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+import torch as th
+import syft as sy
+from syft.frameworks.torch.linalg import inv_sym
+
+
+def test_inv_sym(hook, workers):
+    """
+    Testing inverse of symmetric matrix with MPC
+    """
+    bob = workers["bob"]
+    alice = workers["alice"]
+    crypto_prov = sy.VirtualWorker(hook, id="crypto_prov")
+
+    x = th.Tensor([[0.4627, 0.8224], [0.8224, 2.4084]])
+
+    x = x.fix_precision(precision_fractional=6).share(bob, alice, crypto_provider=crypto_prov)
```

Comment:
Just a curiosity: does is works with precision_fractional=3 ?

Reply:
Yes, but it's less precise (sometimes 0.1 difference from true result)

Reply:
This could be a good use-case for long precision tensors...